browser/js/content.js

```diff
@@ -5,10 +5,51 @@
 const asset = chrome.runtime.getURL;
 
 function init() {
-  // Only do work on reddit.
-  if (!location.hostname.endsWith('.reddit.com')) return;
+  // Passive queries?
+  chrome.storage.sync.get("profile-passive-queries", function(options) {
+    if (options["profile-passive-queries"] !== true) return; // Default false
+
+    if (location.hostname.endsWith('twitter.com')) {
```

Comment:
Not in this case because the [manifest.json only injects it for twitter.com strictly](https://github.com/keybase/client/pull/6943/files#diff-0253589efda58af1677c3d945347dcd5R32). I'm probably being overly broad here but I keep forgetting which sites have other subdomains (like `np.reddit.com` and `www.` and whatever) and which ones don't, so I just try to stick with this more general pattern.